src/content/cloud/personal-access-tokens.mdx

```diff
@@ -11,6 +11,8 @@ You can use Personal Access Tokens instead of OAuth to authenticate with Okteto.
 
 Personal Access Tokens are useful when scripting, or when configuring [Preview Environments](cloud/preview-environments/overview.mdx).
 
+> Tokens are valid for 180 days since their creation. 
```

Comment:
I think we need more than the message.

I think we should add a section on token expiration and explain what happens on expiration, the notification a user will get, and what action is expected. If this is planned on a future doc, please ignore.